documentation/src/main/asciidoc/topics/proc_cli_connecting_container.adoc

```diff
@@ -0,0 +1,31 @@
+[id='cli_start-{context}']
+= Connecting to {brandname} Servers running in a container
```

Comment:
I suggest either:

Establishing a connection with {brandname} running in a container through the CLI  

or

Accessing the CLI of  {brandname}  running in a container